src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -554,6 +554,33 @@ bool clang_cpp_convertert::get_expr(const clang::Stmt &stmt, exprt &new_expr)
     break;
   }
 
+  case clang::Stmt::CXXDeleteExprClass:
+  {
+    const clang::CXXDeleteExpr &de =
+      static_cast<const clang::CXXDeleteExpr &>(stmt);
+
+    if(de.isArrayFormAsWritten())
```

Comment:
I think ternary operator would be better here

Reply:
Done